tower-overload/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "tower-overload"
+version = "0.1.0"
+authors = ["Sean McArthur <sean@seanmonstar.com>"]
```

Comment:
should this be renamed to tower contributors?

Reply:
Heh, that's just a result of `cargo new`. I clicked around, but couldn't find another place where we say "tower contributors". What email is used?

Reply:
At this point, I'm pretty lax about it as we haven't shipped. Before release, I was planning on setting up an email forward. I have `tower-rs.com`, so you can use `team@tower-rs.com` if you want (i have not setup the email forward yet though).